artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -3007,23 +3009,25 @@ private boolean deliver() {
                   logger.trace("Queue " + this.getName() + " is delivering reference " + ref);
                }
 
-               existingMemoryEstimate = ref.getMessageMemoryEstimate();
```

Comment:
Wheres this gone

Reply:
that is just dead code removal, value is not used